worlds/ffmq/Regions.py

```diff
@@ -1,11 +1,9 @@
 from BaseClasses import Region, MultiWorld, Entrance, Location, LocationProgressType, ItemClassification
 from worlds.generic.Rules import add_rule
+from .data.rooms import rooms, entrances
 from .Items import item_groups, yaml_item
-import pkgutil
-import yaml
 
-rooms = yaml.load(pkgutil.get_data(__name__, "data/rooms.yaml"), yaml.Loader)
```

Comment:
I think we should be more thorough in reviews (and hopefully some peer reviewers see this): for yaml this should've been `Utils.parse_yaml`, which is both safer and 7x faster than this. There are very few reasons to use anything else.